flink-java/src/main/java/org/apache/flink/api/java/DataSet.java

```diff
@@ -1377,6 +1377,20 @@ public long count() throws Exception {
 		return new SortPartitionOperator<>(this, field, order, Utils.getCallLocationName());
 	}
 
+	/**
+	 * Locally sorts the partitions of the DataSet on the specified field in the specified order.
+	 * DataSet can be sorted on multiple fields by chaining sortPartition() calls.
```

Comment:
I would not allow chaining of KeySelectors. We only allow a single key extractor anywhere else in the DataSet API (for a good reason). If users want to sort on more than one field, they can return a `TupleX` from the `KeySelector`.